### DISCUSSION TITLE
Getting permission denied when trying to connecting to prod

### DISCUSSION THREAD

**Discussion by oliwerhelsen** [saleor/saleor#7179] [Q&A]
### What I'm trying to achieve
I'm trying to using grapqhl from my local machine to publish new data. But i'm getting permission denied even dough i set all permisson for the app.

Any idea on how to proceed?

**Comment by krzysztofwolski** [comment]
It will be hard to help you without crucial information: 
- which Saleor version are you using?
- examples of requests and responses 

Are you sure you are adding Authorization token ("Bearer XXXX") to the requests?
https://docs.saleor.io/docs/3.0/developer/extending/apps#authentication
